_posts/2022-01-10-reassign-partitions.md

```diff
@@ -0,0 +1,539 @@
+---
+layout: post
+title:  "Reassigning partitions in Apache Kafka Cluster"
+date: 2021-09-23
+author: shubham_rawat
+---
+
+Apache Kafka brokers host replicas of partitions. How many and which partitions each broker replicates affects the load on that broker.
+One broker often ends up experiencing more load than others, and the most loaded broker can become a bottleneck within the cluster.
+Strimzi supports the use of Cruise Control and we recommend using it to balance the load across the brokers.
+That's because when there are more than a handful of partitions in your cluster it becomes very hard to find the best, or even a "good enough", assignment.
+But sometimes you might need to make small changes to the replica assignment.
+
+This blog post will guide you through when you might need to do reassignments using Kafka's partition reassignment rool, and how to do so.
```

Comment:
I would probably change the paragraph structure here. The last sentence of the first paragraph (line 12) looks incomplete without the sentence on line 14 which completes it.

I would either move the sentence from line 12 to form the paragraph with the sentence on line 14. Or join the sentence from line 14 to the initial paragraph.